ui/app/pages/create-account/connect-hardware/account-list.js

```diff
@@ -193,14 +210,54 @@ class AccountList extends Component {
     );
   }
 
+  renderSelectPathPopover() {
+    const { pathValue } = this.state;
+    const { onPathChange } = this.props;
+
+    const footer = (
+      <div className="switch-ledger-path-popover__footer">
+        <Button
+          onClick={() => this.setState({ showPopover: false })}
+          type="secondary"
+          className="invalid-custom-network-alert__footer-row-button"
```

Comment:
This doesn't seem like the appropriate classname :eyes: 

I see it's used for the second button as well